shopfloor_mobile/static/wms/src/scenario/checkout.js

```diff
@@ -281,6 +282,17 @@ const Checkout = {
                 },
             };
         },
+        handle_manual_select_highlight_on_scan: function (res) {
+            const new_data = res.data.select_package;
+            new_data.selected_move_lines.forEach((line) => {
+                const query = $(`[input-value=${line.id}]`);
```

Comment:
Do not rely on the markup. IMO you should simply get the ids of the records to select (or the records themselves) and trigger `manual_select:select_items` and the event handler in manual select will take care of selecting the right items

Reply:
Done.